src/autocomplete/CommandProvider.js

```diff
@@ -2,6 +2,7 @@
 Copyright 2016 Aviral Dasgupta
 Copyright 2017 Vector Creations Ltd
 Copyright 2017 New Vector Ltd
+Copyright 2018 Michael Telatynski <7t3chguy@gmail.com>
```

Comment:
Not sure if this should be New Vector

Reply:
This wasn't a `p1` issue and thus not chargeable hours so `<Me/>`